servlet-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/ee/main/module.xml

```diff
@@ -44,6 +44,7 @@
         <module name="javax.el.api"/>
         <module name="javax.interceptor.api"/>
         <module name="javax.security.jacc.api"/>
+        <module name="javax.transaction.api"/>
```

Comment:
Should this also be optional? Not sure it can, but just curious.

Reply:
It should be optional. The need for it is guarded by a capability check:

https://github.com/wildfly/wildfly/pull/15811/files/53fcd7d2a7a2b16922e240ce26042d26a7a22caf#diff-e73f8c6aa955e801fe07ee3f2e04df349d0b7b7dc3548ad5fe26a3806dd50f95R64

Reply:
fixed